### DISCUSSION TITLE
Unable to build from source

### DISCUSSION THREAD

**Discussion by laurentperez** [rabbitmq/rabbitmq-java-client#723] [Q&A] [answered]
Hello

I'd like to submit a PR to fix a bug, but I can't build from source to pass my tests.

My deps/ folder only contains rabbitmq_codegen and no "rabbit"

Thanks if you can help

```
make: *** /home/lperez/Bureau/work/rabbitmq-java-client/deps/rabbit : Aucun fichier ou dossier de ce type. Arrêt.
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  9.547 s
[INFO] Finished at: 2022-02-09T09:21:35+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.codehaus.gmaven:groovy-maven-plugin:2.1.1:execute (query-test-tls-certs-dir) on project amqp-client: Execution query-test-tls-certs-dir of goal org.codehaus.gmaven:groovy-maven-plugin:2.1.1:execute failed: org.apache.maven.plugin.MojoExecutionException: Failed to query test TLS certs directory with command: make -C /home/lperez/Bureau/work/rabbitmq-java-client/deps/rabbit --no-print-directory show-test-tls-certs-dir DEPS_DIR=/home/lperez/Bureau/work/rabbitmq-java-client/deps -> [Help 1]

```

**Comment by acogoluegnes** [comment]
I will convert this issue to a GitHub discussion. Currently GitHub will [automatically close and lock the issue](https://github.com/github/feedback/discussions/3197) even though your question will be transferred and responded to elsewhere. This is to let you know that we do not intend to ignore this but this is how the current GitHub conversion mechanism makes it seem for the users :(

**Comment by acogoluegnes** [comment, marked as answer]
I re-ran the steps from the README: https://github.com/rabbitmq/rabbitmq-java-client#building-from-source

and it worked, at least to package the JAR file. Which goal did you run?

**Reply by laurentperez** [reply]
I forgot to use the profile exclusion -P '!setup-test-cluster' . It works, thanks !
